packages/components/src/components/tabs/tabs.jsx

```diff
@@ -33,7 +33,7 @@ class Tabs extends Component {
             fit_content,
         } = this.props;
         const { active_index } = this.state;
-        const tab_width = (100 / children.length).toFixed(2);
+        const tab_width = fit_content ? '150px' : `${(100 / children.length).toFixed(2)}%`;
```

Comment:
previous `fit_content` doesn't work(The active line is out of alignment). I make a fix width with `150px` should be sufficient for a a tab title text to fit in.